wordpress-mobile/gutenberg-mobile
# Added list block ordered button e2e test

This change should be merged to `develop` after WordPress/gutenberg#19818 or a similar fix is merged into `develp`, otherwise the new test will (and should) fail. 

### Background
Fixes #1805 
WordPress/gutenberg#19818 fixed a crash on a release branch that was found from pressing the ordered list button on List Block that was caused by a change in Gutenberg (https://github.com/WordPress/gutenberg/pull/19536).

### Description
This change adds the ordered list button press to our Appium tests so any related issues will be caught in the future.

### New Test:
\_\_device-tests__/gutenberg-editor-lists.test.js
_'should be able to add a new List block'_

1. Select List Block.
2. Press any of the controls (i.e: Change the list type to ordered).
3. Verify HTML shows ordered list

### To test:
Run with `yarn test:e2e:ios:local gutenberg-editor-lists.test.js` and `yarn test:e2e:android:local gutenberg-editor-lists.test.js`

*Note: this change should show a failed test on branches that do not yet include the fix from WordPress/gutenberg#19818, and it should pass on branches that have that fix. 

PR submission checklist:
